src/App.js

```diff
@@ -1,14 +1,70 @@
-import React from "react";
-import Form from "./Form";
+import { React, useState } from "react";
 import "./App.css";
 
 function App() {
+  const [numbers, setNumbers] = useState("");
+  const [type, setType] = useState("");
+  const [answer, setAnswer] = useState(null);
+
+  function handleChange(e) {
+    const setMethod = e.target.id === "operation" ? setType : setNumbers;
+    setMethod(e.target.value);
+    console.log("yes i know");
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    const mappedNumbers = numbers.split(",").map((num) => Number(num));
+
+    let textIsValid = mappedNumbers.every((num) => Number.isFinite(num));
+
+    if (!textIsValid || type === "") {
+      setAnswer("Invalid input.");
+    }
+
+    if (type !== "mode") {
+      const sum = mappedNumbers.reduce((first, second) => {
+        return first + second;
+      }, 0);
+
+      type === "sum" ? setAnswer(sum) : setAnswer(sum / mappedNumbers.length);
+    }
+
+    if (type === "mode") {
+      let modeObj = mappedNumbers.reduce((first, second) => {
+        first.hasOwnProperty(second)
+          ? (first[second] += 1)
+          : (first[second] = 1);
+        return first;
+      }, {});
+
+      modeObj.entries();
```

Comment:
`.entries` was used incorrectly here so it is breaking your code. The proper syntax would be `Object.entries(modeObj)` However, it does not look like what entries returned is stored in a variable and therefore not used.